courses/tests.py

```diff
@@ -0,0 +1,22 @@
+from django.test import TestCase
+
+from rest_framework.serializers import ValidationError
+
+from .serializers import JsonListField as JLF
+
+class JsonListFieldTests(TestCase):
+    def test_decodes_string(self):
+        f = JLF()
+        self.assertEquals([], f.to_representation('[]'))
+
+    def test_decodes_unicode(self):
+        f = JLF()
+        self.assertEquals([], f.to_representation(u'[]'))
```

Comment:
In LORE we use `from __future__ import unicode_literals` at the top of each file to specify unicode strings by default, but feel free to leave as is if you think it's preferable the way it is